### PR TITLE
Add smv to workorder tasks

### PR DESCRIPTION
**Pull request by elena-evans**
## Link to JIRA ticket

https://www.pivotaltracker.com/story/show/178775488

## Describe this PR

Added SMV to WorkOrderTask so it can be accessed on the front end

### *What changes have we introduced*

Add descriptions of what changes were made to address the problem and provide a solution.

Include any links to commits that introduce significant additions of code if they help explain the process of coming to the solution. e.g Addition of test database setup, addition of first test and production class, removal of buggy code, etc.

#### _Checklist_

- [ ] Added end-to-end (i.e. integration) tests where necessary e.g to test complete functionality of a newly added feature
- [ ] Added tests to cover all new production code
- [ ] Added comments to the README or updated relevant documentation _(add link to documentation)_, where necessary.
- [ ] Checked all code for possible refactoring
- [ ] Code pipeline builds correctly

### *Follow up actions after merging PR*

Are there any next steps that need to addressed after merging this PR? Add them here.

### CODE REVIEW

File: RepairsApi/V2/Gateways/IRepairsGateway.cs [hunk 1]
```diff
@@ -13,8 +14,8 @@ public interface IRepairsGateway
         Task<int> CreateWorkOrder(WorkOrder raiseRepair);
         Task<IEnumerable<WorkOrder>> GetWorkOrders(IFilter<WorkOrder> filter);
         Task<WorkOrder> GetWorkOrder(int id);
-        Task<IEnumerable<WorkElement>> GetWorkElementsForWorkOrder(WorkOrder workOrder);
-        Task<IEnumerable<WorkElement>> GetWorkElementsForWorkOrder(int id);
+        Task<IEnumerable<WorkOrderTask>> GetWorkElementsForWorkOrder(WorkOrder workOrder);
```
Comment (josh-dadswell-hackney) [inline RepairsApi/V2/Gateways/IRepairsGateway.cs:17]: if this is no longer returning WorkElements can we rename it please

File: RepairsApi/V2/Gateways/RepairsGateway.cs [hunk 2]
```diff
@@ -62,19 +63,42 @@ public async Task<WorkOrder> GetWorkOrder(int id)
             return workOrder;
         }
 
-        public async Task<IEnumerable<WorkElement>> GetWorkElementsForWorkOrder(WorkOrder workOrder)
+        public async Task<IEnumerable<WorkOrderTask>> GetWorkElementsForWorkOrder(WorkOrder workOrder)
```
Comment (josh-dadswell-hackney) [inline RepairsApi/V2/Gateways/RepairsGateway.cs:66]: if this is no longer returning WorkElements can we rename it please

File: RepairsApi/V2/Gateways/RepairsGateway.cs [hunk 3]
```diff
@@ -62,19 +63,42 @@ public async Task<WorkOrder> GetWorkOrder(int id)
...
         {
             return await GetWorkElementsForWorkOrder(workOrder.Id);
         }
 
-        public async Task<IEnumerable<WorkElement>> GetWorkElementsForWorkOrder(int id)
+        public async Task<IEnumerable<WorkOrderTask>> GetWorkElementsForWorkOrder(int id)
```
Comment (josh-dadswell-hackney) [inline RepairsApi/V2/Gateways/RepairsGateway.cs:71]: if this is no longer returning WorkElements can we rename it please

File: RepairsApi/V2/Gateways/RepairsGateway.cs [hunk 4]
```diff
@@ -62,19 +63,42 @@ public async Task<WorkOrder> GetWorkOrder(int id)
...
+            var we =
                 from wo in _repairsContext.WorkOrders.RestrictContractor(_currentUserService)
                 where wo.Id == id
-                select wo.WorkElements;
-
-            return await elements.SingleOrDefaultAsync();
+                from WorkElement in wo.WorkElements
+                select WorkElement;
+
+            if (!we.Any())
```
Comment (Asura5) [inline RepairsApi/V2/Gateways/RepairsGateway.cs:79]: Is there a difference between a work order that isn't found, and a work order that can be found but that has no work elements?

File: RepairsApi.Tests/V2/Gateways/RepairGatewayTests.cs [hunk 5]
```diff
@@ -181,12 +181,12 @@ public async Task CanUpdateWorkStatus()
         }
 
         [Test]
-        public async Task CanGetWorkElements()
+        public async Task CanGetRateScheduleItems()
```
Comment (Asura5) [inline RepairsApi.Tests/V2/Gateways/RepairGatewayTests.cs:184]: could we add a test for the null/empty return please

File: RepairsApi/V2/Gateways/RepairsGateway.cs (hunk 4 above)
Comment (LBHBomotoso) [reply RepairsApi/V2/Gateways/RepairsGateway.cs:79]: Every workorder should have a workelement. Are you saying to search for a work order first?

Comment (Asura5) [reply RepairsApi/V2/Gateways/RepairsGateway.cs:79]: You're right, every work order _should_ have a workelement.

**Review by josh-dadswell-hackney — APPROVED** [review]
